resources/.default-config.edn

```diff
@@ -0,0 +1,5 @@
+{:bucket-name "my-bucket"
+ :bucket-cred {:access-key "xxxxxxxxx"
```

Comment:
If we use amazonica this is redundant, we can pick up creds from the standard places